code/game/gamemodes/cult/cult_items.dm

```diff
@@ -100,12 +105,18 @@
 	icon_state = "cult_hoodalt"
 	item_state = "cult_hoodalt"
 
+/obj/item/clothing/head/culthood/alt/ghost
+	flags = NODROP|DROPDEL
+
 /obj/item/clothing/suit/cultrobes/alt
 	name = "cultist robes"
 	desc = "An armored set of robes worn by the followers of Nar-Sie."
 	icon_state = "cultrobesalt"
 	item_state = "cultrobesalt"
 
+/obj/item/clothing/suit/cultrobes/alt/ghost
```

Comment:
Given how the rune works and that these robes shouldn't be normally obtainable, I can just do that effect to the summoned mob itself.